dv/uvm/core_ibex/Makefile

```diff
@@ -395,12 +394,15 @@ fcov:
           --isa rv32imc \
           --custom_target riscv_dv_extension
 
-# Load verdi to review coverage
-cov_vcs:
-	cd ${OUT}/rtl_sim; verdi -cov -covdir test.vdb &
-
-cov_ius:
-	if [ ! -d "${OUT}/rtl_sim/cov_work/scope/merged_cov" ]; \
-	  then imc -execcmd "merge -out ${OUT}/rtl_sim/cov_work/scope/merged_cov ${OUT}/rtl_sim/cov_work/scope/test_*"; \
-    fi
-	imc -load ${OUT}/rtl_sim/cov_work/scope/merged_cov &
+# Merge coverage in the <out> directory and load into GUI
+cov:
```

Comment:
I think we should split up "merge coverage" and "open a GUI" into two separate steps. Is that possible with VCS's "urg" tool?

If so, this would probably be nicer if the target was a file that gets generated by merging the coverage (`test.vdb`?). Then you add a phony target that depends on that file. See the `$(OUT-SEED)/regr.log` and `post_compare` targets for an example.

Reply:
Ah sorry, I actually meant to remove the `load into GUI` part of this comment, this makefile step is only meant to generate a merged coverage directory. Opening the GUI can be left up to the user, since there are a wide variety of tools out there.

Reply:
Cool, that sounds good to me.